core/email_api.php

```diff
@@ -79,9 +79,16 @@
 # reusable object of class SMTP
 $g_phpMailer = null;
 
-# Indicates whether any emails are currently stored for process during this request.
-# Note: This is only used if not sending emails via cron job
-$g_email_stored = false;
+/**
+ * Indicates how generated emails will be processed by the shutdown function
+ * at the end of the current request's execution; this is a binary flag:
+ * - EMAIL_SHUTDOWN_SKIP       Initial state: do nothing (no generated emails)
```

Comment:
Instead of repeating this list in two places, why don't we document it in the constants file and just refer here to EMAIL_SHUTDOWN_* constants.